app/src/main/java/de/tum/in/tumcampusapp/cards/ChatMessagesCard.java

```diff
@@ -92,7 +92,7 @@ public void setChatRoom(String roomName, int roomId, String roomIdString) {
         mRoomName = mRoomName.replaceAll("\\[[A-Z]+[0-9]+\\]", "");
         mRoomName = mRoomName.trim();
         chatMessageDao.deleteOldEntries();
-        mUnread = chatMessageDao.getLastUnread(roomId);
+        //mUnread = chatMessageDao.getLastUnread(roomId); TODO
```

Comment:
`mUnread` remains uninitialized :boom:

Reply:
Well, there is a todo marker there :tada: